application/values-test.yaml

```diff
@@ -618,4 +618,503 @@ externalSecret:
       annotations: 
         key: value
       labels: 
-        key: value
\ No newline at end of file
+        key: value
+
+# Grafana Dashboard
```

Comment:
@MichaelLi202112 @ufateh as @faizanahmad055 mentioned it will better if we can read them as files; as otherwise the values file will become way too large

Reply:
Added URL approach to specify json content with URL. For file approach, users can create multiple values.yaml files, each of which contains a single dashboard. Providing files for chart to process from parent chart or client side is not supported.